src/scrot.c

```diff
@@ -345,6 +346,24 @@ void scrotDoDelay(void)
     }
 }
 
+static inline long miliToNanoSec(int ms)
+{
+    return ms * 1000L * 1000L;
+}
```

Comment:
`inline` is like `register`: a non-algorithmic optimization that the compiler can trivially figure out for us, and just like `register`, GCC and Clang ignore it entirely. Please remove that.

It only makes sense in library headers due to its semantics but even there I would avoid it.

Reply:
Initially it was in `utils.h` but at some point I moved it back to `scrot.c` since I realized the other awkward ms => ns conversions would go away after using `scrotWaitFor`.

I'll remove the `inline` - I don't like it either.